timeless/sync/synced_customer.py

```diff
@@ -0,0 +1,19 @@
+from timeless.customers.models import Customer
+
+"""
+    Customer synced to database.
+    
+    @todo #225:30min Implement synchronization between Poster and Database for 
+     Customers. Synchronization must be done via celery job. Create tests for 
+     poster Customer retrieval, Customer storage in database and the sync 
+     between the two. See sync of Location implementation as reference. Don't 
+     forget to cover all code added with tests.
+"""
+
+class SyncedCustomer(Customer):
```

Comment:
@paulodamaso I think it's not a good idea to extend not model class from model one. I dont exactly remember the way how `SQL- Alchmemy` will process it, but two options possible:
1) `SQL-Alchemy` will create a One-To-One relation between `SyncedCustomer` and `Customer`
2) `SQL-Alchemy` will create `SyncedCustomer` table inherited from `Customer` (like this https://www.postgresql.org/docs/9.0/ddl-inherit.html)

And looks like both behaviors are wrong for us. So I suggest to use decarator pattern and pass customer instance to contructor. WDYT?